BedrockServer.cpp

```diff
@@ -2020,50 +2001,50 @@ void BedrockServer::_finishPeerCommand(unique_ptr<BedrockCommand>& command) {
 }
 
 void BedrockServer::_acceptSockets() {
-    Socket* s = nullptr;
-    Port* acceptPort = nullptr;
-    while ((s = acceptUnlistedSocket(acceptPort))) {
-        if (SContains(_portPluginMap, acceptPort)) {
-            BedrockPlugin* plugin = _portPluginMap[acceptPort];
-            // Allow the plugin to process this
-            SINFO("Plugin '" << plugin->getName() << "' accepted a socket from '" << s->addr << "'");
-            plugin->onPortAccept(s);
-
-            // Remember that this socket is owned by this plugin.
-            s->data = plugin;
+    // Make a list of ports to accept on.
+    // We'll check the control port, command port, and any plugin ports for new connections.
+    list<reference_wrapper<const unique_ptr<Port>>> portList = {_commandPort, _controlPort};
```

Comment:
This is weird because our ports are `unique_ptr`s, so we can only have one instance of each of them. We can have references to them, but you can't store references in a `std::list`. So, we wrap the references in `reference_wrapper` which is an object that isn't a reference but contains a reference to the underlying `unique_ptr`.